AsyncDisplayKit/Details/UIView+ASConvenience.h

```diff
@@ -42,6 +42,7 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)setNeedsDisplay;
 - (void)setNeedsLayout;
```

Comment:
It's interesting to have these here because they do have special ASDK behavior - so not just normal passthrough properties - however, I think it's good to keep them here for now.  Just an observation, we can later link to a documentation page in a comment to describe the quite amazing layout system :)